### ISSUE TITLE
Stories ignore 

### ISSUE THREAD

**Issue by 5TXdYWK** [telegramdesktop/tdesktop#26852]
### Steps to reproduce

1. open tdesktop for Windows
2. see stories from the Telegram account (or other account)
3. click on profile icon to enlarge it
4. right click the now larger profile photo
5. click on "Hide Stories"
5.1. optionally, also block the profile for good mesaure
6. when a new story is posted by that user, it'll _still_ show up in the main search bar / home screen of the app

### Expected behaviour

Stories from hidden and/or blocked users should not show up at all, regardless of who they are.

### Actual behaviour

Stories from hidden and/or blocked users still show up

![image](https://github.com/telegramdesktop/tdesktop/assets/82390521/10d87322-5881-408f-ae4d-f6f02584356e)

### Operating system

Windows 11

### Version of Telegram Desktop

4.10

### Installation source

Static binary from official website

**Comment by john-preston** [comment]
@5TXdYWK Hiding should work (hide to Archive) in case of other, non Telegram, accounts.

**Comment by 5TXdYWK** [comment]
@john-preston in this case it seems that there's an issue with only the Telegram account, then. It shouldn't be an exception. If a user wants to hide it, it should be hidden.
The "Hide Stories" button currently does nothing for the Telegram account, which is super counterintuitive. (personally i don't want to see _any_ stories, the fact that Telegram can just shrug it away and ignore my preference is pretty unpleasant 😓)

**Comment by LingamYoni** [comment]
i don't want to see **any** stories - please add this option in the settings!

**Comment by all500234765** [comment]
bump

**Comment by mmitkevich** [comment]
please add ignore stories button!
I don't like seing naked women above my work chats )

**Comment by JapstersCavern** [comment]
Can these requests not be treated more seriously guys? (instead of keep trying to let the issue 'lapse' and have an excuse to close it off? - For a start, you have turned Telegram (already MASSIVELY abused by scammers, spammers, 'marketers' and generally unavoidable group/chat, a DM intrusions), into something 100x worse?!

I honestly am pretty appalled by the route that Telegram itself has taken. Stories has COMPLETELY broken Telegram in terms of moderating chats, when this unwanted spam/scam content (Which it really IS, in 99% of cases) can get through REGARDLESS of any chat settings etc?

Introduce a new 'feature' that 1% of legit users use, the other 99% being scammers and spammers? - Is it paying well? - I'm a premium subscriber, and 'stories' is RUINING my chat groups - I only have an option to 'hide' the issue for ME only (So no longer even see them myself, but they'll still flood the chat / group, meaning I'll just let my members get scammed, nice...).

  ...this is NOT how you treat your paying customers guys - I LOVE Telegram, but I HATE stuff like this...

You also owe it to your users' safety to provide a mechanism to disable these in groups - just, please, add a mechanism to DISABLE random 'stories' from popping up in chats, not just 'ignore' them myself??

Plus...  ...It'll certainly earn you a lot of goodwill from the user base, to actually do the RIGHT / ethical thing regarding this experience-ruining issue...

**Comment by 5TXdYWK** [comment]
@JapstersCavern 
> Can these requests not be treated more seriously guys? (instead of keep trying to let the issue 'lapse' and have an excuse to close it off? - For a start, you have turned Telegram (already MASSIVELY abused by scammers, spammers, 'marketers' and generally unavoidable group/chat, a DM intrusions), into something 100x worse?!

It's intentional. They're not interested in improving the application for existing users, they want to push features that will hopefully increase their userbase. It's sad that Telegram has joined in on enshittification.

**Comment by 5TXdYWK** [comment]
By the way, the same applies to other versions of Telegram. The issue for this in Telegram for MacOS was closed, and you can also follow this thread to see how little of a crap Telegram gives about it: https://bugs.telegram.org/c/30207

**Comment by JapstersCavern** [comment]
> By the way, the same applies to other versions of Telegram. The issue for this in Telegram for MacOS was closed, and you can also follow this thread to see how little of a crap Telegram gives about it: https://bugs.telegram.org/c/30207

Yep, I actually think it'll be like UbiSoft and 'The Message' - look what ignoring their user base has done to them - they're screwed.   ie. As regards Telegram, bugs and mass requests have been here, in some cases for YEARS, NOTHING done with them, and they completely ignore us.

They'll probably only react once the userbase gets fed up with being spammed TF by scammers, abusing this wonderful new 'feature'.

As one person in your linked thread mentions - if you are spammed by a service and do not have a method to stop / unsubscribe TOTALLY from it, DM *AND* GROUPS,  then it's akin to being signed up for something and unable to unsubscribe from it - Which is simply NOT acceptable if it's down to the PROVIDER that this is possible, and quite possibly illegal in some regions...

**Comment by LingamYoni** [comment]
Most stories are used to advertise porn-related services. This is a disgrace!

**Comment by JapstersCavern** [comment]
Yep - can't believe this is still apparently the case, and NO response / reassurance / update, or even any acknowledgement of it, from Telegram devs, in this thread - disgraceful, tbh.

Having said that, I'm wondering if someone at TG *has* done something quietly, as in my case at least, this issue seems to have stopped affecting my own groups - strange....  

PS - I've only commented here too, as, like @LingamYoni - I'm NOT going to let this HUGE issue's thread quietly get archived / locked, just to shelve it / benefit TG - I want it actually FIXED for everyone, myself included....  😡

**Comment by Aokromes** [comment]
> Yep - can't believe this is still apparently the case, and NO response / reassurance / update, or even any acknowledgement of it, from Telegram devs, in this thread - disgraceful, tbh.
> 
> Having said that, I'm wondering if someone at TG _has_ done something quietly, as in my case at least, this issue seems to have stopped affecting my own groups - strange....
> 
> PS - I've only commented here too, as, like [@LingamYoni](https://github.com/LingamYoni) - I'm NOT going to let this HUGE issue's thread quietly get archived / locked, just to shelve it / benefit TG - I want it actually FIXED for everyone, myself included.... 😡

https://github.com/telegramdesktop/tdesktop/issues/26852#issuecomment-1732615694 the answer was done the same day the issue was opened.

**Comment by JapstersCavern** [comment]
> > Yep - can't believe this is still apparently the case, and NO response / reassurance / update, or even any acknowledgement of it, from Telegram devs, in this thread - disgraceful, tbh.
> > Having said that, I'm wondering if someone at TG _has_ done something quietly, as in my case at least, this issue seems to have stopped affecting my own groups - strange....
> > PS - I've only commented here too, as, like [@LingamYoni](https://github.com/LingamYoni) - I'm NOT going to let this HUGE issue's thread quietly get archived / locked, just to shelve it / benefit TG - I want it actually FIXED for everyone, myself included.... 😡
> 
> [#26852 (comment)](https://github.com/telegramdesktop/tdesktop/issues/26852#issuecomment-1732615694) the answer was done the same day the issue was opened 1h after it was opened.

NO.  It really wasn't...  @john-preston suggested that I (as a single user) can PERSONALLY hide these (which simply means that *I* as chat owner / random TG user, can no longer see them, but NOT the 1,000's of members in the groups/chats - which is even WORSE, in terms of moderating this crap), and what does "_**other none Telegram accounts**_" even mean?

QUOTE: "**_Hiding should work (hide to Archive) in case of other, non Telegram, accounts._**"

So no, a pointless 'workaround', designed to blinker the chat owner / moderators opting to 'hide' it, is NOT a solution 1hr into the bug being raised...   😡

**Comment by all500234765** [comment]
bump

**Comment by Aokromes** [comment]
If your request asks for modification or removal of platform-wide features (features that work same or similarly in all Telegram apps, both desktop and mobile), then here is not the right place to discuss that. The developmers of Telegram Desktop can't change such things by themselves.

You need to go to https://suggestions.telegram.org to request and/or discuss that and if this will be done in all apps, starting with official mobile apps, which are reference implementations of Telegram platform, then a Desktop-specific request should be opened here.
